app/models/concerns/webhooks/outgoing/event_support.rb

```diff
@@ -27,12 +27,21 @@ def event_type_name
     payload.dig("event_type")
   end
 
-  def endpoints
-    send(BulletTrain::OutgoingWebhooks.parent_association).webhooks_outgoing_endpoints.listening_for_event_type_id(event_type_id)
+  def endpoints(object)
```

Comment:
I think we don't need this `object`, but you can just reference `self` from here.